.env.example

```diff
@@ -42,9 +46,9 @@ TRACECAT__DB_URI=postgresql+psycopg://${TRACECAT__POSTGRES_USER}:${TRACECAT__POS
 NODE_ENV=development
 NEXT_PUBLIC_APP_ENV=development
 # The frontend app URL
-NEXT_PUBLIC_APP_URL=http://localhost:3000
+NEXT_PUBLIC_APP_URL=http://localhost
 # Allows the browser to communicate with the backend
-NEXT_PUBLIC_API_URL=http://localhost:8000
+NEXT_PUBLIC_API_URL=http://localhost/api/
 # Allows the frontend server (inside docker) to communicate with the backend server (inside docker)
 NEXT_SERVER_API_URL=http://api:8000
```

Comment:
Does this need to be changed?

Reply:
Yes because all localhost/api/ paths get redirected to the APi service by caddy.

Reply:
@topher-lo referring to line 49 
`NEXT_SERVER_API_URL=http://api:8000`
or am i missing something here?